style.css

```diff
@@ -210,7 +210,7 @@ button:-moz-focusring,
 [type="button"]:-moz-focusring,
 [type="reset"]:-moz-focusring,
 [type="submit"]:-moz-focusring {
-	outline: 1px dotted ButtonText;
+	outline: 1px dotted buttontext;
```

Comment:
Although not a requirement the suggestion is to maintain CamelCase:
> via https://www.w3.org/TR/CSS22/ui.html#system-colors
> "Although these are case-insensitive, it is recommended that the mixed capitalization shown below be used, to make the names more legible."

WordPress core doesn't use any system colors currently so there's no "prior art" in this regard, I think we should use the more broadly used CamelCase here though.